src/ADAL.PCL.Android/AuthenticationAgentActivity.cs

```diff
@@ -42,6 +42,11 @@ protected override void OnCreate(Bundle bundle)
 
             WebView webView = FindViewById<WebView>(Resource.Id.agentWebView);
             WebSettings webSettings = webView.Settings;
+            string userAgent = webSettings.UserAgentString;
+            webSettings.UserAgentString = 
+                    userAgent + BrokerConstants.ClientTlsNotSupported;
```

Comment:
I don't know why we name this as ClientTlsNotSupport instead of PkeyAuthHeader or something else to make it more clear.